iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/util/IidmXmlUtil.java

```diff
@@ -34,6 +34,13 @@
         }
     }
 
+    public static <C extends AbstractNetworkXmlContext> void assertMaximumVersion(String rootElementName, String elementName, ErrorMessage type, IidmXmlVersion maxVersion, C context) {
```

Comment:
Add javadoc (it should be inspired from `assertMinimumVersion` javadoc)

Reply:
Done.